packages/common/src/groups/HubGroups.ts

```diff
@@ -142,12 +142,14 @@ export async function createHubGroup(
   );
 
   // create or update entity settings
-  const entitySetting = await createOrUpdateEntitySettings(
-    { ...hubGroup, id: entity.id },
-    context.hubRequestOptions
-  );
-  entity.entitySettingsId = entitySetting.id;
-  entity.discussionSettings = entitySetting.settings.discussions;
+  if (!context.isPortal) {
```

Comment:
@dbouwman had specifically requested that the fix for this leverage the hub-permissions subsystem rather than checking `context.isPortal`. I _believe_ we could call `checkPermission` with the `hub:group:settings:discussions` permission for this.